### DISCUSSION TITLE
github not login

### DISCUSSION THREAD

**Discussion by huihu233** [vercel/community#3264] [Help] [answered]
### Summary

Unable to log in using github account

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/59477091/90e5cb25-b799-4896-a5f5-05e6f0235986)

**Comment by swarnava** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username used for account creation. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
